Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# syntax=docker/dockerfile:1
+
+######################### BUILD STAGE 1 ############################
+# This stage builds the HarbourBridge binary using its source code.
+
+# Add the official Docker golang image as the base image for the build stage
+FROM golang:latest AS build
+
+# Set the default work directory as “/hbBin” in the container.
+WORKDIR /hbBin
+
+# Copy HarbourBrdige’s source code into the working directory.
+COPY . ./
```

Comment:
Are we sure this is what the command is supposed to do?  this looks like it is copying current directory to current directory. 
Also typo in in comment `HarbourBrdige’s`